### PR TITLE
Add suse support (#2)

### PR DESCRIPTION
**Pull request by jvega10**
Adding Attestation support for SUSE Linux

### CODE REVIEW

File: tdx-attestation-app/package/azguestattestation1-1.0.3-2.x86_64.rpm
Comment (jianlia) [inline tdx-attestation-app/package/azguestattestation1-1.0.3-2.x86_64.rpm:1]: Do we still need this?

Comment (jvega10) [reply tdx-attestation-app/package/azguestattestation1-1.0.3-2.x86_64.rpm:1]: good catch, no only the one that is being used in the suse_install, let me make the change

**Review by jianlia — APPROVED** [review]

File: tdx-attestation-app/suse_install.sh [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+# Install required packages
+sudo zypper install -y -t pattern devel_basis
+sudo zypper install -y cmake
+sudo zypper install -y nlohmann_json-devel
+sudo zypper install -y libcurl4
+sudo zypper install -y libcurl-devel
+sudo zypper install -y libboost_headers1_75_0-devel
+
+# Install GuestAttestation Debian Package
+sudo rpm --nosignature -i package/azguestattestation1-1.0.5-2.x86_64.rpm
```
Comment (canfikret) [inline tdx-attestation-app/suse_install.sh:19]: Please don't submit rpm files; they are large and can get quickly outdated.

You can do rpm -ihv https://package.microsoft.com/...../azguestattestation1-1.0.5-2.x86_64.rpm

File: tdx-attestation-app/suse_install.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+# Update
+sudo zypper --gpg-auto-import-keys refresh
+
+# Install required packages
+sudo zypper install -y -t pattern devel_basis
+sudo zypper install -y cmake
+sudo zypper install -y nlohmann_json-devel
+sudo zypper install -y libcurl4
+sudo zypper install -y libcurl-devel
+sudo zypper install -y libboost_headers1_75_0-devel
```
Comment (canfikret) [inline tdx-attestation-app/suse_install.sh:16]: libssl-dev is needed?

**Review by canfikret — CHANGES_REQUESTED** [review]

File: tdx-attestation-app/suse_install.sh (hunk 1 above)
Comment (jvega10) [reply tdx-attestation-app/suse_install.sh:19]: This is only for private preview. Those packages are official release, is that correct?

File: tdx-attestation-app/suse_install.sh (hunk 2 above)
Comment (jvega10) [reply tdx-attestation-app/suse_install.sh:16]: SUSE Linux uses different libraries and this version of the packages I am using has openssl statically link

File: tdx-attestation-app/suse_install.sh (hunk 1 above)
Comment (jvega10) [reply tdx-attestation-app/suse_install.sh:19]: Additionally, this change is not going to main branch. Before this goes to main branch, we will release package as you mentioned above!

**Pull request merged**